{
marker_color_ranges: {
  yellow: {
    r_min: 0.37747,
    r_max: 0.45,
    g_min: 0.382739,
    g_max: 0.452026,
    a_min: 0.6,
    a_max: 0.699346,
    r_avg: 0.413958,
    g_avg: 0.412759,
    a_avg: 0.654129
  },
  green: {
    r_min: 0.195531,
    r_max: 0.292191,
    g_min: 0.421801,
    g_max: 0.502793,
    a_min: 0.456209,
    a_max: 0.564706,
    r_avg: 0.242011,
    g_avg: 0.45923,
    a_avg: 0.517936
  },
  blue: {
    r_min: 0,
    r_max: 0.00297619,
    g_min: 0.359673,
    g_max: 0.402402,
    a_min: 0.435294,
    a_max: 0.479739,
    r_avg: 0.000155901,
    g_avg: 0.379324,
    a_avg: 0.456693
  }
},
color_tolerance: 0.07,
alpha_tolerance: 0.15,
cluster_markers: true
}
